src/Adapters/Auth/utils.js

```diff
@@ -0,0 +1,8 @@
+exports.isJSON = string => {
```

Comment:
Move this into `src/Utils.js`. We already have a `isPromise` function there. Function name camelcase `isJson`

Reply:
Fixed